src/common.ts

```diff
@@ -12,6 +12,8 @@ export type FilePath = string;
 /** Type alias to indicate the path of a directory. */
 export type DirPath = string;
 
+export type PoolMetadataSource = 'none' | 'direct' | { smashUrl: string };
```

Comment:
This should have a doc comment like all the other types, and should go in the cardanoWallet module.